stacks/services/APIGateway.ts

```diff
@@ -0,0 +1,96 @@
+import {
+    AuthorizationType,
+    AwsIntegration,
+    MethodOptions,
+    PassthroughBehavior,
+    Resource,
+} from "aws-cdk-lib/aws-apigateway";
+import { PolicyStatement, Role, ServicePrincipal } from "aws-cdk-lib/aws-iam";
+import { ApiGatewayV1Api, Stack, Bucket } from "sst/constructs";
+import { SiriGeneratorStack } from "../SiriGeneratorStack";
+
+export const createSiriApi = (stack: Stack, siriSXBucket: Bucket): void => {
+    const apiGateway = new ApiGatewayV1Api(stack, "SiriSXApi", {
+        cdk: {
+            restApi: {
+                restApiName: `SiriSXApi-${stack.stage}`,
```

Comment:
Go with the naming convention of `cdd-siri-sx-api-${stage}` here

Reply:
done